src/app/views/services-offered/services-offered.component.html

```diff
@@ -1 +1,25 @@
-<p>services-offered works!</p>
+<p>Below is just to demonstrate how the tabbed card works. Still needs style work.</p>
```

Comment:
I appreciate you doing this as Tyler asked.

I would push back though against the idea of putting just this into it's own component. The design for a universal component, in my understanding, is that the content of the component is determined as it's used. So the user may want multiple tabbed-cards within the application. Would they then make a specific component as a sort of wrapper for each one, and then just have the content for that specific card like how you have it here?

I think you could definitely argue yes, it really depends on the application. I personally wouldn't do it this way. I would just find whatever component represents that specific page, and then put all this html in in there. 

Note: I wouldn't just shove this all into the app.component (as it was originally), that was just for demonstration.